### PR TITLE
Beginnings of Git Branching Tutorial

### PR DESCRIPTION
**Pull request by am0eba-byte**
I didn't want to go ahead and add my new HTML page to the /docs directory just in case things exploded, but I did change the link to the git/github tutorials link in the main index.html

### CODE REVIEW

**Review by ebeshero — APPROVED** [review]
@am0eba-byte Looks good to me! Maybe let's leave this PR open for a moment to talk through our workflow process this evening.

File: docs/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1>New Tutorials and Projects In Progress for Newtfire</h1>
         <nav>
             <ul class="menuB">
                 <li><a href="index.html">Home</a></li>
-                <li>Git / GitHub</li>
```
Comment (Tiny-Pickles) [inline docs/index.html:14]: looking good!

**Comment by ebeshero** [comment]
@am0eba-byte Just to give you another excuse to update your branch, I'm going to merge this PR now!

[esc] :wq  Is the key sequence to get out of vim. 
Typing "i" gets you in insert mode so you can use your arrow keys and move around vim.
Here's a fun and practical article about vim: https://opensource.com/article/19/3/getting-started-vim

**Comment by ebeshero** [comment]
Leaving your branch open. :-)

**Pull request merged**